From 7e43a635dfeec81e2b7d1872813bd5e7d79512c9 Mon Sep 17 00:00:00 2001
From: Wadamzmail <alksndrmetwakil@gmail.com>
Date: Sun, 19 Jul 2026 08:20:13 +0200
Subject: [PATCH] libx11-depend-on-android-shmem

---
 packages/libx11/build.sh | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/packages/libx11/build.sh b/packages/libx11/build.sh
index 863edca..d6d341a 100644
--- a/packages/libx11/build.sh
+++ b/packages/libx11/build.sh
@@ -11,7 +11,7 @@ TERMUX_PKG_SHA256=69606f485c2c07c14ef64f75b7bb326d48587af33795d9ab3e607c0b5f94f1
 TERMUX_PKG_AUTO_UPDATE=true
 # building all reverse dependencies of libx11 requires xorgproto
 # and the x11.pc file of libx11 explicitly marks a dependency on xorgproto
-TERMUX_PKG_DEPENDS="libandroid-support, libxcb, xorgproto"
+TERMUX_PKG_DEPENDS="libandroid-support, libandroid-shmem, libxcb, xorgproto"
 TERMUX_PKG_BUILD_DEPENDS="xorg-util-macros, xtrans"
 TERMUX_PKG_RECOMMENDS="xorg-xauth"
 TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
@@ -19,6 +19,10 @@ ac_cv_path_RAWCPP=/usr/bin/cpp
 --enable-malloc0returnsnull
 "
 
+termux_step_pre_configure() {
+       LDFLAGS+=" -landroid-shmem"
+}
+
 termux_step_post_massage() {
 	# Regression test for broken XLC_LOCALE files. Do not remove.
 	local f
-- 
2.52.0

